datahub-web-react/src/app/entity/dataProcessInstance/preview/Preview.tsx

```diff
@@ -39,10 +39,10 @@ export const Preview = ({
     health,
     parentEntities,
     parentContainers,
-}: // duration,
-// status,
-// startTime,
-{
+    duration,
```

Comment:
@asikowitz we're adding some of the properties that are specific to data process instance in preview -- let me know if this makes sense